docs/openapi-ts/migrating.md

```diff
@@ -54,19 +54,50 @@ This config option is deprecated and will be removed.
 
 ### Single `enums.gen.ts` file
 
-Enums are now exported from a single file. If you used imports from `model.ts`, you can change it to `enums.gen.ts`.
+Enums are now exported from a separate file. If you use imports from `models.ts`, you can change them to `enums.gen.ts`.
```

Comment:
Should we make this models.gen.ts

Reply:
No because until this release, there was no `models.gen.ts`